features/document-colors.yml

```
@@ -0,0 +1,15 @@
+name: Document colors
+description: A document's `bgColor`, `fgColor`, `alinkColor`, `linkColor`, and `vlinkColor` properties get and set the background, foreground, active link, link, and visited link colors of that document.
+spec: https://html.spec.whatwg.org/multipage/obsolete.html
+discouraged:
+  according_to:
+    - https://html.spec.whatwg.org/multipage/obsolete.html
+  alternatives:
+    - background-color
+    - color
```

```suggestion
    - color
    - css-object-model
```
I think this would be a good alternative as well, as it is how you would manipulate the document's styles. Potentially also `get-computed-style` as an entry point to that?

---

```suggestion
    - color
    - css-object-model
    - get-computed-style
```

Agreed! Thanks.